integration-tests/src/tests/client/process_blocks.rs

```diff
@@ -2828,14 +2828,8 @@ fn test_epoch_multi_protocol_version_change() {
     let mut seen_v2 = false;
 
     let mut height = 1;
-    while chrono::Utc::now() < end_time {
-        let head = env.clients[0].chain.head().unwrap();
-        let epoch_id = env.clients[0]
-            .epoch_manager
-            .get_epoch_id_from_prev_block(&head.last_block_hash)
-            .unwrap();
-        let protocol_version =
-            env.clients[0].epoch_manager.get_epoch_protocol_version(&epoch_id).unwrap();
+    while chrono::Utc::now() < end_time && (!seen_v0 || !seen_v1 || !seen_v2) {
```

Comment:
Nice, was there a visible speed up in the execution time of this test?

Reply:
Yeah decent speedup locally, from 6s down to 2.5s